examples/step-4/doc/intro.dox

```diff
@@ -90,7 +90,7 @@ The same can be done with member variables. Consider the following
 function, which might in turn call the above one:
 @code
   template <int dim>
-  void make_grid_and_dofs (Triangulation<dim> &triangulation)
+  void make_grid (Triangulation<dim> &triangulation)
```

Comment:
Actually, this isn't right. The text here reads like this:
```
The same can be done with member variables. Consider the following
function, which might in turn call the above one:
@code
  template <int dim>
  void make_grid_and_dofs (Triangulation<dim> &triangulation)
  {
    make_grid (triangulation);

    DoFHandler<dim> dof_handler(triangulation);
    ...
  };
@endcode
```
The function name should really be as originally stated.

Reply:
Ah. I concur. We can just close this one, and not merge.